packages/dm-core/src/components/UiPluginSelector.tsx

```diff
@@ -149,24 +144,25 @@ function filterPlugins(
 }
 
 export function UIPluginSelector(props: {
-  absoluteDottedId?: string
+  idReference: string
   type: string
   onSubmit?: (data: any) => void
   categories?: string[]
   breadcrumb?: boolean
   referencedBy?: string
   onOpen?: (data: any) => void
+  config?: any
 }): JSX.Element {
   const {
-    absoluteDottedId,
+    idReference,
     type,
     categories,
     breadcrumb,
     referencedBy,
     onSubmit,
     onOpen,
   } = props
-  const { uiRecipes, loadingBlueprint, error } = useBlueprint(type)
+  const { uiRecipes, isLoading, error } = useBlueprint(type)
```

Comment:
suggestion:
rename loading to isContextLoading
and isLoading to isBlueprintLoading